### DISCUSSION TITLE
Please my project by your favorite TV shows

### DISCUSSION THREAD

**Discussion by tkmaeda** [NAIST-SE/AttractingContributors#39] [General]
I need to fill in at least two more items in my list!

==============================
I need your help to complete my project! 

Link to my project:

https://github.com/maedatak/AttractingContributors-TVShows

You can check these links for ideas:

https://www.netflix.com/jp-en/browse/genre/83
https://www.hulu.com/hub/tv

**Comment by tattiyaS** [comment]
I added a show to your list.

**Comment by vittunyuta-aom** [comment]
added!!

**Comment by noppadol-assava** [comment]
I commit pull request on your repo.
Please help me adding Food into the list.
https://github.com/NoppadolAssava/AttractingContributors---Food

**Comment by nosfill** [comment]
I submitted a PR.

**Comment by satokibi44** [comment]
added!
Cloud you contribute my repo?
https://github.com/satokibi44/AttractingContributors-Anime

**Reply by tkmaeda** [reply]
You already merged my PR :)

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors
